docs/source/isaura.rst

```diff
@@ -1,5 +1,163 @@
+
+.. _Isaura display:
+
+ .. image:: images/isaura/r8250_evt194237_dhits.png
+   :width: 46%
+ .. image:: images/isaura/r8250_evt194237_voxels.png
+   :width: 53%
+
+
+Since events have already been divided into voxel-made 3D tracks, the next (and final) step comprises the computation of the **blobs** [#]_. This is a crucial stage within the reconstruction chain, since the energy of these elements will allow us to distinguish between double-electron (such as the double-beta signal) and single-electron (the majority of backgrounds) tracks.
+
+To define the center of the blobs, we use the following procedure, as illustrated in the figure below. First, the two extreme voxels of the track are localized following the BFS algorithm. The energy-weighted averaged position of the hits inside the voxel will correspond to the **blob center**. From that point, a 3D sphere of radius ``blob_radius`` (specified in the config file) is taken, and all hits inside that sphere will contribute to the energy of the blob.
+
+ .. image:: images/isaura/blobs_position_definition.png
+   :width: 400
+   :align: center
+
+
+Thus, to carry out this procedure, the first thing to do is to localize the two end voxels for each track. Defining the distance between any pair of voxels as the shortest path **along the track** that connects them, the two extreme voxels will be the ones with the longest distance between them. However, there are two special cases that are important to comment:
+
+
+ - It is possible that some spurious **low-energy** hits appear around the track (due to over-iterations during the Richarson-Lucy deconvolution process, as commented in :doc:`beersheba`; or some noise inside the chamber, for example). If these hits are reconstructed around the track but not far enough to produce a different S2 or track (taking into account the voxel size), they can be considered as a part of the main one and, being a bit separate, it is probable that they end up belonging to an extreme voxel. That case would not be correct, and in order to solve it, the voxel will be dropped from the track and its energy passed to the closest one. This process is only carried out if the voxel energy is lower than: ``energy_threshold`` and the track is made by more than ``min_voxels`` voxels. Once this procedure is done, the extreme voxels are searched and found again recursively, until none of these conditions are fulfilled.
```

Comment:
`is lower than:`. I think that `:` is a typo, right?

Reply:
ok